hello.py

```diff
@@ -1,5 +1,18 @@
 #!python
 # test file to make sure your env is working
 import sys
-v = sys.version_info
-print(f"Hello from python {v[0]}.{v[1]}.{v[2]}!")
+import libsql_client as libsql
+
+statement = """insert into recipe (title, description, ingredients, instructions, difficulty_level, image, food_type, recipe_source)
+                                values ("","","","","","","","")
+                                """
+
+url = "libsql://recipes-deparr.turso.io"
+auth_token = "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9.eyJpYXQiOiIyMDIzLTEwLTIwVDE4OjMzOjQyLjc0OTA1NjA1OVoiLCJpZCI6ImViMTdhZDMxLTZhZWYtMTFlZS05NzA3LWZhMThmYjJkYTBjNSJ9.p2Kh1iEmdl0Y8LYYJnD-pfZWj5hXNqFOnOTFcSerWSsqBUPTVXm6c0KbqUVCK8JXL8cEoc83NZ9IUWt4RJBWBw"
```

Comment:
This token is no longer valid, and I think gitguardian wont let us merge this until the token and url are removed.